app/javascript/components/FilterSlider.js

```diff
@@ -107,26 +107,26 @@ export default function FilterSlider(props) {
     <>
       <input
         id='input-min-organism-age'
-        onChange={(event) => onNumberInputChange(event)}
+        onChange={event => onNumberInputChange(event)}
         type="number"
         min={min}
         max={max}
         value={inputValues[0]}
-        style={{'width': '60px'}}
+        style={{ 'width': '60px' }}
       />
-      <span style={{'margin': '0 4px 0 4px'}}>-</span>
+      <span style={{ 'margin': '0 4px 0 4px' }}>-</span>
       <input
         id='input-max-organism-age'
-        onChange={(event) => onNumberInputChange(event)}
+        onChange={event => onNumberInputChange(event)}
         type='number'
         min={min}
         max={max}
         value={inputValues[1]}
-        style={{'width': '60px', 'marginRight': '8px'}}
+        style={{ 'width': '60px', 'marginRight': '8px' }}
       />
-      <select value={unit} onChange={(event) => {updateUnit(event.target.value)}}>
+      <select value={unit} onChange={event => {updateUnit(event.target.value)}}>
         {units.map((unit, i) =>
```

Comment:
I don't understand where there is a comma added to the end of this line -- is that a stray keystroke, or a style rule?

Reply:
It was a style rule -- the formatter automatically inserted trailing commas.  I removed that linter rule and this trailing comma.